wgpu-core/src/device/mod.rs

```diff
@@ -1266,6 +1278,119 @@ impl<B: GfxBackend> Device<B> {
         })
     }
 
+    fn create_buffer_descriptor<'a>(
```

Comment:
This method has too many arguments causing a clippy error. Not sure the best way to clean this up.

Reply:
in the worst case, just add the `[allow(...)]` on it